### ISSUE TITLE
Visual Studio 2017 not detected, even though it is installed

### ISSUE THREAD

**Issue by Yatayata** [playscale/stashed.io#22]
I have installed Visual Studio 2017, but the tool is unable to find it. I have reinstalled Visual Studio 2017, but no luck.

I am using version1.4.6.

![aum41onh90](https://user-images.githubusercontent.com/701085/41759855-7d98c908-762b-11e8-9244-61390cc9c7f7.png)

**Comment by jsmouret** [comment]
Hello,

Can you try running those lines in a command prompt?

```
cd "C:\Program Files (x86)\Microsoft Visual Studio\Installer"

vswhere -all -prerelease -products * -version [15.0,16.0) -requires Microsoft.VisualStudio.Component.VC.Tools.x86.x64 -format json
```

If the output is empty, you might be missing C++ Tools.

Thank you

**Comment by Yatayata** [comment]
It's not empty:

![output](https://user-images.githubusercontent.com/701085/41760520-d1df06f0-762e-11e8-9b9e-8aabd1715ccf.png)

**Comment by jsmouret** [comment]
Damn, that's strange...
Can I see a screenshot of the front end tab in the dashboard?
I'm curious to see if it is listing the right Visual Studio version at least...

**Comment by Yatayata** [comment]
![front](https://user-images.githubusercontent.com/701085/41762883-3a0f05ce-7637-11e8-83b3-f306bb5de543.png)

**Comment by Yatayata** [comment]
Hi, any update on this?

My colleague installed the software on his PC, but he has the same issue with VS2017.

**Comment by Justin-Randall** [comment]
Thanks for the output. It will make it easier to reproduce then include in the tests for the next release. No ETA just yet, but will try to include this fix.

**Comment by dibben-jsol** [comment]
I confirmed that in my case the problem is the "description:" field in the json output.

I downloaded the source for vswhere from github (https://github.com/Microsoft/vswhere), commented out the line that writes the description to the json output and compiled a new version. When that version of vswhere is placed in the Installer folder instead of the original then stashed.io finds VS2017 OK.

It is not a very nice workaround, but it does mean I can test stashed.io with VS2017.
